File: aula06/atividade02/Pessoa.java
package atividade02;

public class Pessoa {
    String origem;
    int idade;
    String genero;
    String cor;
    String profissao;

    public Pessoa(String origem, int idade, String genero, String cor, String profissao) {
        this.origem = origem;
        this.idade = idade;
        this.genero = genero;
        this.cor = cor;
        this.profissao = profissao;
    }

    public String apresentacao() {
        return "\n Origem: " + this.origem + "\n Idade: " + this.idade + "\n Genero: " + this.genero + "\n Cor: " + this.cor + "\n Profissão: " + this.profissao + "\n";
    }

    public static void main(String[] args) {
        Pessoa pessoa = new Pessoa("pobre", 35, "masculino", "branco", "desenvolvedor");

        System.out.println(pessoa.apresentacao());
    }
}

File: aula06/atividade01/Pessoa.py
class Pessoa():
    def __init__(self, origem, idade, genero, cor, profissao) -> None:
        self.origem = origem
        self.idade = idade
        self.genero = genero
        self.cor = cor
        self. profissao = profissao
        
    def __str__(self) -> str:
        return f'\n Origem: {self.origem}\n Idade: {self.idade}\n Genero: {self.genero}\n Cor: {self.cor}\n Profissão: {self.profissao}\n'

pessoa = Pessoa('humilde', 35, 'masculino', 'branco', 'Desenvolvedor')
print(pessoa)
